front50-core/src/main/java/com/netflix/spinnaker/front50/model/pipeline/Pipeline.java

```diff
@@ -1,133 +0,0 @@
-/*
- * Copyright 2016 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- */
-package com.netflix.spinnaker.front50.model.pipeline;
-
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.common.base.Strings;
-import com.netflix.spinnaker.front50.model.Timestamped;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-
-public class Pipeline extends HashMap<String, Object> implements Timestamped {
-
-  private static ObjectMapper MAPPER = new ObjectMapper();
-  public static final String TYPE_TEMPLATED = "templatedPipeline";
-
-  @JsonIgnore
-  public String getApplication() {
-    return (String) super.get("application");
-  }
-
-  @JsonIgnore
```

Comment:
@ajordens leaving a comment so you can view the `@JsonIgnore` annotations in the original Pipeline model in front50-core.